en/advanced_config/sensor_thermal_calibration.md

```diff
@@ -35,8 +35,10 @@ PX4 supports two calibration procedures:
 * [onboard](#onboard_calibration) - calibration is run on the board itself. This method requires knowledge of the amount of temperature rise that is achievable with the test setup.
 * [offboard](#offboard_calibration) - compensation parameters are calculated on a development computer based on log information collected during the calibration procedure. This method allows users to visually check the quality of the data and curve-fit.
 
-The offboard approach is more complex and slower, but requires less knowledge of the test setup and is easier to validate. 
+The offboard approach is more complex and slower, but requires less knowledge of the test setup and is easier to validate.
 
+> **Note** After thermal calibration the thermal calibration parameters (`TC_*`) are used for *all* calibration/compensation of the respective sensors.
+  The normal calibration parameters (e.g. `SYS_CAL_*`) are no longer used and are reset to their default values.
 
```

Comment:
It is correct that the calibration is a one time thing and the SYS_CAL_* parms specify the boundaries used in this process. So those params will not be used anymore after the caliibration has completed. But if I remember correctly only the boolean values are set back to 0. The SYS_CAL_TDEL, SYS_CAL_TMAX, SYS_CAL_TMIN params stay the way they have been set for the calibration.

Reply:
Thanks. I modified this line to "The normal `SYS_CAL_*` calibration parameters are no longer used (and in some cases may be reset)."

I didn't want to be precise without actually testing.